source/model/breadcrumbs.ts

```diff
@@ -0,0 +1,65 @@
+const typeManual = 'manual';
```

Comment:
It looks like we didn't apply prettier styles in this file :(

Reply:
I ran `npm run lint` and didn't see any change. @konraddysput what is incorrect here?